src/common/SIRF.py

```diff
@@ -535,13 +555,17 @@ def unary(self, f, out=None):
 
     @property
     def shape(self):
-        '''Returns the shape of the data array
+        '''Returns the shape of the object data
         '''
+        if self.is_empty():
+            return None
```

Comment:
maybe to be more numpy-like...

```suggestion
            return (0,)
```

Reply:
fine for me. Best to add this to CHANGES then